DIGITAL_OCEAN_README.md

```diff
@@ -0,0 +1,62 @@
+# Digital Ocean Setup
+
+### Gotchas that will save you time:
+
+* Create a project in Google Cloud Provider (GCP). Copy the project ID for the next step.
```

Comment:
This is true for each provider that has a non-default backend (e.g. see the [`backend.tf`](https://github.com/w3f/polkadot-validator-setup/blob/master/terraform/azure/backend.tf) file for Azure). Maybe it's worth to include these general instructions in `GUIDE_COMPLETE.md` briefly.
Also, if you just want to test it locally without GCP you can always use `terraform init -backend=false`

Reply:
Sure, I'll add that to the GUIDE_COMPLETE 👍🏼